server/models/buoy.py

```diff
@@ -22,3 +23,5 @@ class Buoy(Base):
     water_quality = Column(String(1), default="n", nullable=False)
     dart = Column(String(1), default="n", nullable=False)
     seq = Column(SmallInteger, nullable=True)  # tao_seq
+
+    coastlines = relationship("Coastline", back_populates="buoy", uselist=True)
```

Comment:
I like this use of SQLAlchemy 2.0 syntax. It's great to back populate tables that are related to each other. Really puts the relationships in relational databases.